configargparse.py

```diff
@@ -607,10 +607,6 @@ def get_items_for_config_file_output(self, source_to_settings,
                         if value is not None:
                             if isinstance(value, bool):
                                 value = str(value).lower()
-                            elif callable(action.type):
```

Comment:
This code was added to handle custom types (https://github.com/bw2/ConfigArgParse/issues/69)
Wouldn't taking it out break that again?

Reply:
Never mind, just read your comments on the new issue.